src/NzbDrone.Core/Notifications/Mailgun/MailgunSettings.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Notifications.Mailgun
+{
+    public class MailGunSettingsValidator : AbstractValidator<MailgunSettings>
+    {
+        public MailGunSettingsValidator()
+        {
+            RuleFor(c => c.ApiKey).NotEmpty();
+            RuleFor(c => c.From).NotEmpty();
+            RuleFor(c => c.Recipients).NotEmpty();
+        }
+    }
+
+    public class MailgunSettings : IProviderConfig
+    {
+      private static readonly  MailGunSettingsValidator Validator = new MailGunSettingsValidator();
+
+      public MailgunSettings()
+      {
+          Recipients = new string[] { };
+      }
+
+      [FieldDefinition(0, Label = "API Key", HelpText = "The API key generated from MailGun")]
+      public string ApiKey { get; set; }
+
+      [FieldDefinition(1, Label = "Use EU Endpoint?", HelpText = "You can choose to use the EU MailGun endpoint with this", Type = FieldType.Checkbox)]
```

Comment:
This help text seems weird. I know it came from Sonarr, but feels like it could be better... maybe without the “You can”?